github/Repository.py

```diff
@@ -1599,7 +1629,37 @@ def create_secret(self, secret_name, unencrypted_value):
         status, headers, data = self._requester.requestJson(
             "PUT", f"{self.url}/actions/secrets/{secret_name}", input=put_parameters
         )
-        return status == 201
+        return github.Secret.Secret(self._requester, headers, data, completed=True)
+
+    def get_variables(self):
+        """
+        Gets all organization variables
+        :rtype: List[github.Variable.Variable]
+        """
+        headers, data = self._requester.requestJsonAndCheck("GET", f"{self.url}/actions/variables")
+        return [
+            github.Variable.Variable(
+                requester=self._requester,
+                headers={},
+                attributes={"url": f"{self.url}/actions/variables/{variable['name']}"},
+                completed=False,
+            )
+            for variable in data["variables"]
+        ]
+
+    def get_variable(self, variable_name: str):
+        """
+        :calls: 'GET /orgs/{org}/actions/variables/{variable_name} <https://docs.github.com/en/rest/actions/variables#get-an-organization-variable>`_
+        :param variable_name: string
+        :rtype: github.Variable.Variable
+        """
+        assert isinstance(variable_name, str), variable_name
+        return github.Variable.Variable(
+            requester=self._requester,
+            headers={},
+            attributes={"url": f"{self.url}/actions/variables/{variable_name}"},
+            completed=False,
+        )
 
     def create_variable(self, variable_name: str, value: str) -> bool:
         """
```

Comment:
Please update the urls, here from

    https://docs.github.com/en/rest/reference/actions/variables#create-a-repository-variable

to

    https://docs.github.com/en/rest/actions/variables#create-a-repository-variable